README.md

```diff
@@ -20,6 +20,20 @@ By default, \*.cljc files are identified as Clojure files. Thus when you eval, i
 Clojure file. If you'd like to eval as a ClojureScript file, run the command `Editor: Set current
 editor syntax`, select `ClojureScript` and then eval.
 
+## Clojure(script) error messages
+Currently this plugin supports an experimental feature that allows to change the default exception behavior (showing the complete stacktrace) to an expandable version of the same. Some screenshots of the new behavior can be found [here](https://github.com/LightTable/Clojure/issues/77).
+To use the new behavior go to ```User behaviors``` and add the following lines :
+
+```Clojure
+; for Clojure
+[:editor.clj :-lt.plugins.clojure/clj-exception]
+[:editor.clj :lt.plugins.clojure/clj-expandable-exception]
```

Comment:
I don't understand what you mean. You mean to add the collapsible-exception object itself?

I already added the `expandable-exceptions` behavior in `clojure.behaviors` which reacts to the `:editor.exception.collapsible`. That behavior will be hooked to the clojure editor by default but it will never trigger as it is only raised by the `clj-expandable-exception` behavior which the users has to enable. That was per your specifications, or did I misunderstood what you mean?